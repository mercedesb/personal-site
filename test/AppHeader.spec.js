// import { shallowMount, createLocalVue } from '@vue/test-utils'
// import AppHeader from '@/components/AppHeader.vue'

// describe('AppHeader', () => {


//   const initialProps = {
//     classModifier: 'modifier'
//   }

//   const shallow = propsData => shallowMount(AppHeader, { 
//     propsData: {
//       ...initialProps,
//       ...propsData
//     }
//    })

//   describe('Snapshots', () => {
//     let component 

//     describe('with a classModifier passed in', () => {
//       it('matches snapshot', () => {
//         component = shallow()
//         expect(component.element).toMatchSnapshot()
//       })
//     })

//     describe('without a classModifier passed in', () => {
//       it('matches snapshot', () => {
//         component = shallow({classModifier: ''})
//         expect(component.element).toMatchSnapshot()
//       })
//     })
//   })

//   describe('Properties', () => {
//     let component 

//     beforeEach(() => {
//       component = shallow()
//     })

//     it('has a classModifier property', () => {
//       expect(component.props().classModifier).toBe('modifier')
//     })
//   })

//   describe('Computed', () => {
//     let component

//   })
// })