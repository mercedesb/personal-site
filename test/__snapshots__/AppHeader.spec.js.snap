// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader Snapshots with a classModifier passed in matches snapshot 1`] = `
<nav
  class="MainNav"
>
  <div
    class="FlexContainer FlexContainer--alignCenter FlexContainer--modifier"
  >
    <div
      class="MainNav-home"
    >
      <router-link
        to="/"
      >
        <span
          class="MainNav-logo"
        >
          Mercedes Bernard
        </span>
      </router-link>
       
      <!---->
    </div>
     
    <div>
      <div
        class="MainNav-hamburger"
      >
        <img
          alt="hamburger menu"
          src="../assets/menu.svg"
        />
      </div>
       
      <!---->
    </div>
  </div>
</nav>
`;

exports[`AppHeader Snapshots without a classModifier passed in matches snapshot 1`] = `
<nav
  class="MainNav"
>
  <div
    class="FlexContainer FlexContainer--alignCenter FlexContainer--undefined"
  >
    <div
      class="MainNav-home"
    >
      <router-link
        to="/"
      >
        <span
          class="MainNav-logo"
        >
          Mercedes Bernard
        </span>
      </router-link>
       
      <!---->
    </div>
     
    <div>
      <div
        class="MainNav-hamburger"
      >
        <img
          alt="hamburger menu"
          src="../assets/menu.svg"
        />
      </div>
       
      <!---->
    </div>
  </div>
</nav>
`;
